site/en/docs/extensions/mv3/content_scripts/index.md

```diff
@@ -152,14 +152,14 @@ They can include JavaScript files, CSS files or both. All auto-run content scrip
 
 #### Exclude matches and globs {: #matchAndGlob }
 
 Specified page matching is customizable by including the following fields in the manifest
 registration.
 
-<table class="simple"><tbody><tr><th>Name</th><th>Type</th><th>Description</th></tr><tr id="exclude_matches"><td><code>exclude_matches</code></td><td>array of strings</td><td><em>Optional.</em> Excludes pages that this content script would otherwise be injected into. See <a href="match_patterns">Match Patterns</a> for more details on the syntax of these strings.</td></tr><tr id="include_globs"><td><code>include_globs</code></td><td>array of strings</td><td><em>Optional.</em> Applied after <code>matches</code> to include only those URLs that also match this glob. Intended to emulate the <a href="http://wiki.greasespot.net/Metadata_Block#.40include"><code>@include</code> </a>Greasemonkey keyword.</td></tr><tr id="exclude_globs"><td><code>exclude_globs</code></td><td>array of string</td><td><em>Optional.</em> Applied after <code>matches</code> to exclude URLs that match this glob. Intended to emulate the <a href="http://wiki.greasespot.net/Metadata_Block#.40include"><code>@exclude</code> </a>Greasemonkey keyword.</td></tr></tbody></table>
+<table class="simple"><tbody><tr><th>Name</th><th>Type</th><th>Description</th></tr><tr id="exclude_matches"><td><code>exclude_matches</code></td><td>array of strings</td><td><em>Optional.</em> Excludes pages that this content script would otherwise be injected into. See <a href="/docs/extensions/mv3/match_patterns/">Match Patterns</a> for more details on the syntax of these strings.</td></tr><tr id="include_globs"><td><code>include_globs</code></td><td>array of strings</td><td><em>Optional.</em> Applied after <code>matches</code> to include only those URLs that also match this glob. Intended to emulate the <a href="http://wiki.greasespot.net/Metadata_Block#.40include"><code>@include</code> </a>Greasemonkey keyword.</td></tr><tr id="exclude_globs"><td><code>exclude_globs</code></td><td>array of string</td><td><em>Optional.</em> Applied after <code>matches</code> to exclude URLs that match this glob. Intended to emulate the <a href="http://wiki.greasespot.net/Metadata_Block#.40include"><code>@exclude</code> </a>Greasemonkey keyword.</td></tr></tbody></table>
```

Comment:
```suggestion
<table class="simple"><tbody><tr><th>Name</th><th>Type</th><th>Description</th></tr><tr id="exclude_matches"><td><code>exclude_matches</code></td><td>array of strings</td><td><em>Optional.</em> Excludes pages that this content script would otherwise be injected into. See <a href="/docs/extensions/mv3/match_patterns/">Match Patterns</a> for more details on the syntax of these strings.</td></tr><tr id="include_globs"><td><code>include_globs</code></td><td>array of strings</td><td><em>Optional.</em> Applied after <code>matches</code> to include only those URLs that also match this glob. Intended to emulate the <a href="http://wiki.greasespot.net/Metadata_Block#.40include"><code>@include</code> </a>Greasemonkey keyword.</td></tr><tr id="exclude_globs"><td><code>exclude_globs</code></td><td>array of string</td><td><em>Optional.</em> Applied after <code>matches</code> to exclude URLs that match this glob. Intended to emulate the <a href="http://wiki.greasespot.net/Metadata_Block#.40exclude"><code>@exclude</code> </a>Greasemonkey keyword.</td></tr></tbody></table>
```

Update `@exclude` link to point to the right section in the linked document.